Userland/Games/2048/Game.cpp

```diff
@@ -161,7 +163,16 @@ static bool is_stalled(const Game::Board& board)
     return true;
 }
 
-Game::MoveOutcome Game::attempt_move(Direction direction)
+static size_t get_number_of_free_cells(const Game::Board& board)
+{
+    size_t accumulator = 0;
+    for (auto& row : board)
+        for (auto& cell : row)
+            accumulator += cell == 0;
```

Comment:
Please use curly braces for if/for bodies with more than one line.

```suggestion
    for (auto& row : board) {
        for (auto& cell : row)
            accumulator += cell == 0;
    }
```